charts/netbox/values.schema.json

```diff
@@ -1060,7 +1060,7 @@
               "type": "boolean"
             },
             "mirrorGroupsExcept": {
-              "type": "string"
+              "type": "array"
```

Comment:
```suggestion
              "type": ["null", "string", "array"]
```

Reply:
https://django-auth-ldap.readthedocs.io/en/latest/reference.html#auth-ldap-mirror-groups-except says, this has to be an Array or None. So when would it be a String?